mypyc/genops.py

```diff
@@ -421,6 +421,9 @@ def prepare_class_def(module_name: str, cdef: ClassDef, mapper: Mapper) -> None:
     ir.mro = mro
     ir.base_mro = base_mro
 
+    for base in ir.mro[1:]:
+        base.has_children = True
```

Comment:
I would write this as `base.children.append(ir)`, so that we can do this (later) also for classes with _few_ (like 2-3) subclasses.